Uno.Gallery/Uno.Gallery.Shared/Views/NestedPages/MediaPlayerElementSample_NestedPage4.xaml

```diff
@@ -59,8 +60,8 @@
 														IsFastForwardEnabled="{Binding IsChecked, ElementName=IsFastForwardEnabled}"
 														IsFastRewindButtonVisible="{Binding IsChecked, ElementName=IsFastRewindButtonVisible}"
 														IsFastRewindEnabled="{Binding IsChecked, ElementName=IsFastRewindEnabled}"
-														IsFullWindowButtonVisible="{Binding IsChecked, ElementName=IsFullWindowButtonVisible}"
-														IsFullWindowEnabled="{Binding IsChecked, ElementName=IsFullWindowEnabled}"
+														not_win:IsFullWindowButtonVisible="{Binding IsChecked, ElementName=IsFullWindowButtonVisible}"
+														not_win:IsFullWindowEnabled="{Binding IsChecked, ElementName=IsFullWindowEnabled}"
```

Comment:
if you are going to comment out no-existing properties for windows
make sure you also do the same for the option buttons controlling them